### DISCUSSION TITLE
Add Kutt

### DISCUSSION THREAD

**Discussion by Datenschmutz** [coollabsio/coolify#4362] [Service Template Requests]
Kutt is a modern URL shortener with support for custom domains. Shorten URLs, manage your links and view the click rate statistics.

https://kutt.it
https://github.com/thedevs-network/kutt

**Comment by docmedicus-support** [comment]
I would like to second the suggestion, since Shlink is great, but does not really have a viable multi-user frontend at the moment (until Shlink Dashboard is released).

**Comment by softzer0** [comment]
I would like to see this natively supported in Coolify, too.
